tests/stp/stripe/run.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+cd $HOME/TorQ/deploy/TorQ/latest
+source setenv.sh
+
+# Handle command-line arguments
+source $KDBTESTS/flagparse.sh
+
+# Path to test directory
+testpath=${KDBTESTS}/stp/stripe
+export perfpath=${testpath}/performance
+
+# Start procs
+${TORQHOME}/torq.sh start discovery1 stp1 rdb1 rdb2 rdb3 rdb4 -csv ${testpath}/process.csv
+
+# Start test proc
+QCMD='taskset -c 0,1 /usr/bin/rlwrap q'
+${QCMD} ${TORQHOME}/torq.q \
+  -proctype test -procname test1 \
+  -test ${testpath} \
+  -load ${KDBTESTS}/helperfunctions.q ${testpath}/settings.q \
+  -testresults ${KDBTESTS}/stp/results/stripe/ \
+  -runtime $run \
+  -procfile ${testpath}/process.csv \
+  -write
+  #$debug $stop $write $quiet
+
+# k4unit test
+k4unit=$(find "$(cd ..; pwd)" -name "k4unit.q")
+# Performance test from stp
+cd $perfpath
+# Specify session name
+session=kdbProcess
+# Kill remaining session first
+tmux kill-session -t $session
```

Comment:
is it typical in other TorQ tests to run things in a tmux (I haven't checked other tests)? not necessarily a problem, only issue is tmux isn't necessarily installed by default on all linux servers so it seems maybe a little strange to rely on it to run the tests

Reply:
I had to run the performance test on the stp and when I tried to run the test using the torq.q method, the rdbs are not able to connect to it before the test runs. I.e. the test runs on the stp port before the starting up the stp in debug mode. As a result, all the test runs and fail before it starts up in debug mode. Only then the connections to the rdbs are made. 

Which is why I had to rely on such an unorthodox method to run the performance test on the stp itself. Do you have any suggestion on how I can maybe change it?

Reply:
Ah OK, I see why this is an issue... I'll have a think about it and see if I can think of any suggestions that could make this a bit simpler etc.